article/app/views/liveblog/liveBlogBodyContentAMP.scala.html

```diff
@@ -28,6 +29,11 @@
         @if(model.currentPage.currentPage.isArchivePage) {
             @fragments.liveBlogNavigation(article.content.metadata.id, model.currentPage, amp = true)
         }
+
+            <script type="application/ld+json">
```

Comment:
Why wouldn't this go into https://github.com/guardian/frontend/blob/master/common/app/views/fragments/metaData.scala.html instead of in the template for the live blog?